src/nexpy/gui/plotview.py

```diff
@@ -112,20 +111,19 @@ def new_figure_manager(label=None, *args, **kwargs):
     label : str
         The label used to define 
     """
-    import matplotlib.pyplot as plt
```

Comment:
you don't actuallly need `global` here as you are mutating it (not re-assigning it)

Reply:
Thanks - good to know.